Relax version constraints to work with Stackage LTS 19.

Taken from upstream:

  https://github.com/ganeti/ganeti/commit/4f8d61ea0101721eae1c6f43be8430d819e5e611

diff --git a/cabal/ganeti.template.cabal b/cabal/ganeti.template.cabal
index bb4ff8053..98491dd9f 100644
--- a/cabal/ganeti.template.cabal
+++ b/cabal/ganeti.template.cabal
@@ -63,14 +63,14 @@ library
     , unix                          >= 2.5.1.0
     , utf8-string                   >= 0.3.7
 
-    , attoparsec                    >= 0.10.1.1   && < 0.14
-    , base64-bytestring             >= 1.0.0.1    && < 1.2
+    , attoparsec                    >= 0.10.1.1   && < 0.15
+    , base64-bytestring             >= 1.0.0.1    && < 1.3
     , case-insensitive              >= 0.4.0.1    && < 1.3
     , curl                          >= 1.3.7      && < 1.4
     , hinotify                      >= 0.3.2      && < 0.5
     , hslogger                      >= 1.1.4      && < 1.4
     , json                          >= 0.5        && < 1.0
-    , lens                          >= 3.10       && < 5.0
+    , lens                          >= 3.10       && < 6.0
     , lifted-base                   >= 0.2.0.3    && < 0.3
     , monad-control                 >= 0.3.1.3    && < 1.1
     , parallel                      >= 3.2.0.2    && < 3.3
